ci/build_go.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+RAPIDS_VERSION="$(rapids-version)"
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file-key go \
```

Comment:
do you know how I could add this file key to `rapids-dependency-file-generator`?

Reply:
cc @bdice @vyasr  @AyodeAwe any suggestions here?

Reply:
Add a key under `files:` in `dependencies.yaml`, then add dependency lists with the conda packages you need. Here’s the rust one for comparison: https://github.com/rapidsai/cuvs/blob/710e9f5a541c518deffb91f75a87cd4fe1372a8a/dependencies.yaml#L70

Reply:
More explanation in the README here: https://github.com/rapidsai/dependency-file-generator